Language/tr/Database.php

```diff
@@ -11,8 +11,21 @@
 
 // Database language settings
 return [
-    'invalidEvent'         => '{0} geçersiz Model Event geri çağrısı.',
-    'invalidArgument'      => 'Geçerli bir {0} belirtmelisiniz.',
-    'invalidAllowedFields' => 'Modelde izin verilen alanlar belirtilmeli: {0}',
-    'emptyDataset'         => '{0} içinde veri yok.',
+    'invalidEvent'                     => '"{0}" geçerli bir Model Olayı (Event) geri çağırma işlevi değil.',
+    'invalidArgument'                  => 'Geçerli bir "{0}" sağlamalısınız.',
+    'invalidAllowedFields'             => 'İzin verilen alanlar model için belirtilmeli: "{0}"',
+    'emptyDataset'                     => '{0} için veri yok.',
+    'emptyPrimaryKey'                  => '{0} yapmaya çalışırken birincil anahtar tanımlı değil.',
+    'failGetFieldData'                 => 'Veritabanından alan verileri alınamadı.',
```

Comment:
'Veritabanından alan verileri alınamadı.',
should be
'Veri tabanından alan verileri alınamadı.',
All "veritabanı" words should be written separately as "veri tabanı".